src/FondOfSpryker/Yves/GoogleTagManager/Plugin/VariableBuilder/TransactionProductVariables/TransactionProductFieldBrandPlugin.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace FondOfSpryker\Yves\GoogleTagManager\Plugin\VariableBuilder\TransactionProductVariables;
+
+use Exception;
+use FondOfSpryker\Yves\GoogleTagManager\Dependency\VariableBuilder\TransactionProductFieldPluginInterface;
+use Generated\Shared\Transfer\GooleTagManagerTransactionProductTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Spryker\Shared\Log\LoggerTrait;
+use Spryker\Yves\Kernel\AbstractPlugin;
+
+/**
+ * @method \FondOfSpryker\Yves\GoogleTagManager\GoogleTagManagerFactory getFactory()
+ */
+class TransactionProductFieldBrandPlugin extends AbstractPlugin implements TransactionProductFieldPluginInterface
+{
+    use LoggerTrait;
+
+    public const ATTR_BRAND = 'brand';
+    public const DEFAULT_LOCALE = '_';
```

Comment:
Ist das hier richtig mit dem Underscore?

Reply:
Jupp, wundert mich auch jedes mal wieder. Underscore ist unlocalized, in den localizedAttributes gibt es die Felder nicht.